packages/venia-concept/.buildpack/create.js

```diff
@@ -0,0 +1,175 @@
+const { resolve } = require('path');
+function createProjectFromVenia(fse) {
+    const gitIgnoredGlob = fse
+        .readFileSync(resolve(__dirname, '../.gitignore'), 'utf-8')
+        .trim()
+        .split('\n')
+        .map(line => {
+            try {
+                if (
+                    fse.statSync(resolve(__dirname, '..', line)).isDirectory()
+                ) {
+                    return line + '{/*,/**/*}';
+                }
+            } catch (e) {}
+            return line;
+        })
+        .join(',');
+    const allIgnoredGlob = `{CHANGELOG*,LICENSE*,.buildpack/*,${gitIgnoredGlob}}`;
+
+    const toCopyFromPackageJson = [
+        'main',
+        'browser',
+        'dependencies',
+        'devDependencies',
+        'optionalDependencies',
+        'engines'
+    ];
+    const scriptsToCopy = [
```

Comment:
I think it would be better to just remove the unnecessary commands. AFAICT these are the excluded commands:

* `build:dev` - I think this is a valid command, should remain so that you can build a dev bundle for whatever reason.
* `load-env` - not sure if this is necessary anymore.
* `now-build` - another thing I don't think is necessary
*  `stats` - can remove entirely. Usage is included in `build:analyze`.
* `storybook`, `storybook:build` both are deprecated. Can probably just delete them from the scripts.